### PR TITLE
IDE-362 - CI/CD

### PR DESCRIPTION
**Pull request by ofairfoul**

### CODE REVIEW

File: .github/workflows/release-docker-service.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: Build and release backend docker application
```
Comment (h4xnoodle) [inline .github/workflows/release-docker-service.yml:1]: I'm not really a fan of the name of this one, as it suggests we are 'releasing' or 'deploying' something, when it's just uploading an image that looks like it's just used as a cache

Comment (ofairfoul) [reply .github/workflows/release-docker-service.yml:1]: yeah, i kinda followed the standard which FSF and BEF started. i'm also not a fan of it

Comment (ofairfoul) [reply .github/workflows/release-docker-service.yml:1]: its not just a cache its the released image version. 
when you want to deploy the release, you pull and push it to the repo.
that way even if you destroyed your ECR, you would still have the image in GHCR.

**Review by h4xnoodle — APPROVED** [review]
LGTM, let's use these and iterate

**Pull request merged**